main.cc

```diff
@@ -1111,6 +1112,14 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 forward_service.stop().get();
             });
 
+            distributed<service::tablet_allocator> the_tablet_allocator;
```

Comment:
nit: The `the_` prefix is in use by `debug` namespace to store global pointers to services for scylla-gdb.py and whoever tries to debug scylla needs. So this `the_` prefix in local variable name looks confusing

Reply:
Ok, I'll change it.